README.md

```diff
@@ -82,84 +78,21 @@ The installation prerequisites are the same as described in the TensorFlow
 
 #### Installation
 
-1. Once TensorFlow's dependencies are installed, clone the source of the 
-   [tensorflow] repo to your machine. 
-
-     :warning: You need the following version of TensorFlow: `v1.12.0`
-
-        git clone https://github.com/tensorflow/tensorflow.git
-        cd tensorflow
-        git checkout v1.12.0
-        git status
-        HEAD detached at v1.12.0
-   
-2. You must instantiate a specific kind of `virtualenv`  to be able to proceed 
-   with the `ngraph-tf` bridge installation. For systems with Python 3.n or 
-   Python 2.7, these commands are
-
-        virtualenv --system-site-packages -p python3 your_virtualenv 
-        virtualenv --system-site-packages -p /usr/bin/python2 your_virtualenv  
-        source your_virtualenv/bin/activate # bash, sh, ksh, or zsh
-        
-   Note: Depending on specific version of the Python and components already
-   installed on your system - the list of dependent Python components vary. 
-   You may need to install one or more of the following Python packages: `numpy mock keras keras_applications protobuf keras_preprocessing`.
-   
-3. Now run `./configure` and choose `no` for the following when prompted to build TensorFlow.
-
-    XLA support:
-
-        Do you wish to build TensorFlow with XLA JIT support? [Y/n]: n
-        No XLA JIT support will be enabled for TensorFlow.
-
-    CUDA support:
-    
-        Do you wish to build TensorFlow with CUDA support? [y/N]: N
-        No CUDA support will be enabled for TensorFlow.
-    
-    Note that if you are running TensorFlow on a Skylake family processor then select
-    `-march=broadwell` when prompted to specify the optimization flags:
-    
-        Please specify optimization flags to use during compilation 
-        when bazel option "--config=opt" is specified 
-        [Default is -march=native]: -march=broadwell
-    
-    This is due to an issue in TensorFlow tracked in this issue: 
-    https://github.com/tensorflow/tensorflow/issues/17273
-
-4. Prepare the pip package and the TensorFlow C++ library:
+1. Once TensorFlow's dependencies are installed, clone `ngraph-tf` repo:
 
-        bazel build --config=opt //tensorflow/tools/pip_package:build_pip_package
-        bazel-bin/tensorflow/tools/pip_package/build_pip_package ./
-
-5. Install the pip package, replacing the `tensorflow-1.*` with your 
-   version of TensorFlow:
-
-        pip install -U ./tensorflow-1.*whl
-   
-6. Now clone the `ngraph-tf` repo one level above -- in the 
-  *parent* directory of the `tensorflow` repo cloned in step 1:
-
-        cd ..
         git clone https://github.com/NervanaSystems/ngraph-tf.git
         cd ngraph-tf
         git checkout v0.8.0
```

Comment:
There is no tag for v0.8.0
See this error when i checkout 
```
git checkout v0.8.0
```
error: pathspec 'v0.8.0' did not match any file(s) known to git.